easybuild/easyblocks/g/gromacs.py

```diff
@@ -480,13 +533,16 @@ def sanity_check_step(self):
         lib_files = []
         bin_files = []
 
-        for configopts in configopts_list:
-            # add the _d suffix to the suffix, in case of the double precission
-            if re.search('DGMX_DOUBLE=(ON|YES|TRUE|Y|[1-9])', configopts, re.I):
-                suff = '_d'
+        if self.cfg['double_precision'] is None or self.cfg['double_precision']:
```

Comment:
You are updating configopts if this is set, don't you? So This check is not required. Removing it catches user-set -DGMX_DOUBLE